compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -2169,11 +2169,11 @@ object Parsers {
                 // Don't error in non-strict mode, as the alternative syntax "implicit (x: T) => ... "
                 // is not supported by Scala2.x
               report.errorOrMigrationWarning(
-                s"This syntax is no longer supported; parameter needs to be enclosed in (...)",
+                s"This syntax is no longer supported; parameter needs to be enclosed in (...)${rewriteNotice()}",
                 in.sourcePos())
```

Comment:
It would be clearer if the position surrounded the whole thing that needs to be put in parens, e.g.:
```suggestion
                Span(start, in.lastOffset))
```